style.css

```diff
@@ -0,0 +1,323 @@
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #fff;
+  width: 100%;
+}
+
+.buto {
+  border: none;
+  background-color: #fff;
+  padding: 23px 23px 10px 10px;
+}
+
+.logo {
+  font-family: "poppins", sans-serif;
+  color: #6070ff;
+  font-weight: bold;
+  font-size: large;
+  padding: 20px 10px 10px 25px;
+  text-decoration: none;
+}
+
+.intro {
+  font-family: "poppins", sans-serif;
+  color: #172b4d;
+  width: 60%;
+  font-weight: 900;
+  font-size: 18px;
+}
+
+.introduction {
+  font-family: "poppins", sans-serif;
+  color: #344563;
+  font-weight: 400;
+  font-size: 16px;
+  width: 100%;
+  margin-top: 0;
+}
+
+.social ul {
+  display: flex;
+  gap: 15px;
+  list-style: none;
+  padding: 0;
+}
+
+.scroll {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 30px;
+  width: 100%;
+  padding: 100px 0;
+  background-color: #f7f7f9;
+}
+
+.box-contain {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #7a869a;
+  width: 90%;
+  margin: 0 auto;
+  border-radius: 16px;
+  background-color: #fff;
+  list-style: none;
+  margin-bottom: 50px;
+}
+
+.box-contain ul {
+  padding: 0;
+  margin: 0;
+}
+
+.ptflio-item {
+  width: 90%;
+  margin: 0 auto;
+  list-style: none;
+  padding: 20px 0;
+  background-color: #fff;
+  display: flex;
+  flex-flow: column;
+}
+
+.ptflio-image {
+  height: 220px;
+  width: 100%;
+  left: 16px;
+  top: 16px;
+  border-radius: 8px;
+}
+
+.lists-container {
+  width: 100%;
+  align-items: center;
+}
+
+.lists {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 80%;
+}
+
+.lists-item {
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  text-transform: capitalize;
+}
+
+.lists .obect1 {
+  text-transform: capitalize;
+  color: #344563;
+}
+
+.text-holder {
+  position: static;
+  width: 327px;
+  left: 24px;
+  top: 276px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  padding-top: 8px;
+  padding-bottom: 6px;
+}
+
+.helloo {
+  position: static;
+  width: 327px;
+  height: 146px;
+  left: 24px;
+  top: 276px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  padding-top: 8px;
+  padding-bottom: 6px;
+}
+
+.language-container {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  list-style: none;
+}
+
+.language-item {
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 5px 10px;
+  margin: 0 2px;
+  font-size: 14px;
+  border-color: #396df2;
+  color: #396df2;
+}
+
+.butto {
+  font-size: 19px;
+  color: #396df2;
+  padding: 13px;
+  background-color: transparent;
+  margin-top: 2rem;
+  border: 1px solid #396df2;
+  border-radius: 6px;
+}
+
```

Comment:
- [OPTIONAL ] Although the previous reviewer did not see this but please make the  `See project`  `button` follow the button styling: Active, Hover, and Pressed (see design)
![Screenshot from 2022-12-03 19-12-27](https://user-images.githubusercontent.com/92303891/205457943-227f0990-7771-43ac-9c9c-dc7f6cdd5c46.png)

Reply:
- [OPTIONAL] - Please kindly implement the previous code reviewer's suggestion, even it's optional, about buttons effects such as `Enabled`, `Hover`, `Pressed`, and `Disabled`